doc/changes/unreleased.md

```diff
@@ -18,15 +19,42 @@ organization by reading through ``doc/user_guide/encryption.rst``.
 2. If needed, update your usage of ``pyexasol.connect(...)`` & ``ExaConnection`` to 
 reflect your organization's needs.
 
+### IMPORT & EXPORT
+With Exasol DB versions x & x, the default behavior has changed. Previously, the default
```

Comment:
Need details updated regarding version impact - is it known or who to get this info from?